spec/cc/engine/csslint_spec.rb

```diff
@@ -16,6 +16,11 @@ module Engine
           expect{ lint.run }.to output(/Don't use IDs in selectors./).to_stdout
         end
 
+        it 'fails on malformed file' do
+          create_source_file('foo.css', '')
```

Comment:
Is empty really "malformed"? I think I'd either update the docstring to say "fails on empty file" or update the example to put some malformed content in `foo.css`.